### ISSUE TITLE
IN的track_running_state是否应该设置为True

### ISSUE THREAD

**Issue by SherryYu33** [Andong-Li-speech/TaylorSENet#1]
因为默认是False，是不是应该手动设置一下True

**Comment by Andong-Li-speech** [comment]
> 

Yes, and IN will use the utterance-level statistics (mean, var) by default. Here we set it as False so the system is actually not a strict causal SE and you can switch it into True or use other norms (iLN, cLN, BN) to guarantee the causality and we have tested them, and cLN is highly suggested as the performance is relatively better than iLN and BN.

**Comment by Andong-Li-speech** [comment]
> 因为默认是False，是不是应该手动设置一下True

Yes, and IN will use the utterance-level statistics (mean, var) by default. Here we set it as False so the system is actually not a strict causal SE and you can switch it into True or use other norms (iLN, cLN, BN) to guarantee the causality and we have tested them, and cLN is highly suggested as the performance is relatively better than iLN and BN.

**Comment by SherryYu33** [comment]
您好，我在模型中把IN的track_running_state设为了True，在推理时尝试了两种方式

1.model.eval() 
with torch.no_grad()

Average WB-PESQ: 2.190
Average NB-PESQ: 2.903
Average STOI: 91.167
Average SISNR: 12.363

2.仅用了with torch.no_grad()

Average WB-PESQ: 2.837
Average NB-PESQ: 3.336
Average STOI: 96.108
Average SISNR: 16.751

第二种方式的结果比第一种好了很多，这是不是因为第二种计算方式是非因果的？

**Comment by malingling0510** [comment]
> 因为默认是False，是不是应该手动设置一下True

您好，我想问一下，您在调试的时候使用的数据集是从 DNS-challenge官网上下载的嘛？
